### ISSUE TITLE
Alien pawns do not show Overall Armor in Gear tab

### ISSUE THREAD

**Issue by Fireflywater** [AthenaSulisMinerva/CombatExtendedFastTrack#281]
## Basic Information
**Operating System:** Windows 10 64-bit
**Physical Memory/RAM Capacity:** 16 GB

**RimWorld Version:** 1.0.2408 rev747
**CE FastTrack Version:** 1.0.1.8-FT-014
**Affected Mod(s) and Version(s):** 
```
* Kurin, The Three Tailed Fox - Ver 1.0.0.0
* Rabbie the Moonrabbit race - Ver Steam: Oct 17, 2019
```

**Summary of Issue:**
Alien races do not show Overall Armor in the Gear tab. Simple as that. Humans are as expected. 
While Kurin and Rabbie was only tested, it is highly likely that this is an issue with other Alien races.

**Screenshots:**
```
![image](https://user-images.githubusercontent.com/51966765/73714401-a2ad7600-46c5-11ea-9899-d0c9232adba9.png)
```

## Declarations of Due Diligence
- [x] (Tick) By submitting this bug report, I declare that:

* I have tested to make sure a new savegame works with only Core and CE:FT
* I have tested to make sure this issue is not also found in the original Combat Extended mod
* I have narrowed down the affected mod(s), by testing them individually or through a binary search
* I have searched the issue tracker, and made sure this issue hasn't been reported before
* I am reporting only one issue at a time, and not combining several issues in one report
* I understand that the CE:FT team reserves the right to close my issue at any time, regardless of whether or not my problem has been resolved, if I have been found to not have performed the due diligence required

**Comment by Fireflywater** [comment]
I have done some digging in the source material and have come to the likely conclusion that a simple <patch> XML will not work and that there's an issue about the hardcoding referring to Human. Though this would not explain.

**Comment by AthenaSulisMinerva** [comment]
I'm definitely seeing this on my end as well, for various other factions that use the Humanoid Alien Races framework.

**Comment by meatball113** [comment]
Yeah I've noticed this issue when I used the main CE + alien races
In my current CE:FT playthrough Kurin, Arachne, Racc, Rabbie do not have the overall armor resists in gear tab (although it seems their resists are functional, just not displaying in the tab.)

The O21 races, goblins, elves are working correctly and displaying information

**Comment by AthenaSulisMinerva** [comment]
> The O21 races, goblins, elves are working correctly and displaying information

This is an interesting observation, @meatball113!

The O21 races also make use of Humanoid Alien Races framework, which suggests there is a difference in how the race patches for the other non-O21 mods were implemented.

I'll take a look over the next few days if I have time.
